Packs/IP2LocationIO/Integrations/IP2LocationIO/IP2LocationIO.py

```diff
@@ -0,0 +1,126 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+import urllib3
+from typing import Any, Dict, List
+
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def get_ip_geolocation(self, ip: str, api_key: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix='/',
+            params={
+                'ip': ip,
+                'key': api_key
+            }
+        )
+
+
+def test_module(client: Client) -> str:
+    try:
+        return client._http_request(
+            method='GET',
+            url_suffix='/',
+            params={
+                'ip': '8.8.8.8'
+            }
+        )
```

Comment:
```suggestion
        client._http_request(
            method='GET',
            url_suffix='/',
            params={
                'ip': '8.8.8.8'
            }
        )
```
This way, if the HTTP request fails (raises an exception) the test module will fail, and if not, we will return an 'ok', as we do not interested in the response of the request itself, only in its success